### DISCUSSION TITLE
How to add attributes to li tag when extending bullet list extension?

### DISCUSSION THREAD

**Discussion by mcnaveen** [ueberdosis/tiptap#3306] [General Questions]
I'm trying to extend the bullet list extension with drag functionality. Right now the whole `<ul>` block is draggable. But I want to extend it even further for each `<li>` element.

I have the below code and can able to add attributes to `<ul>` tags.

But I'm not sure how to add attributes to the `<li>` tag. Because I need to add `draggable` and `data-drag-handle` to it.

![image](https://user-images.githubusercontent.com/8493007/195549930-f2d5a5f1-9495-464a-a8e0-86d1186216c5.png)

**Comment by Nantris** [comment]
This would be a very welcome addition.
